docs/registry/advanced/_index.md

```diff
@@ -20,7 +20,8 @@ aliases:
   - "/modular-resources/advanced/"
 ---
 
-Some use cases may require you to define a new API, or deploy a custom component using a server on a {{< glossary_tooltip term_id="remote" text="remote part" >}}.
+Some use cases may require advanced considerations when designing or deploying modular resources.
+Depending on your needs, you may wish to define a new API subtype, deploy a custom component using a server on a {{< glossary_tooltip term_id="remote" text="remote part" >}}, or design a custom ML model.
```

Comment:
```suggestion
Depending on your needs, you may wish to define a new API subtype, deploy a custom component using a server on a {{< glossary_tooltip term_id="remote" text="remote" >}} {{< glossary_tooltip term_id="part" text="part" >}}, or design a custom ML model.
```
Not sure if you want to include this tooltip, I thought it could be nice.